README.md

```diff
@@ -32,6 +32,57 @@ To uninstall TinyTeX, use the command:
 choco uninstall tinytex
 ```
 
-## Scoop package
+## Scoop packages
 
-Coming...
+Scoop is another package manager for Windows. You need to install [_scoop_](https://scoop-docs.now.sh/docs/getting-started/Quick-Start.html) first to use it from powershell.
+
+Apps for _scoop_ are available through "buckets". For now, TinyTex binary packages are not available in the default **main** or **extras** buckets that comes with a new installation of _scoop_, but through the scoop bucket **r-bucket** at https://github.com/cderv/r-bucket/ with other R-related tools.
+
+You need first to add this bucket
+```powershell 
+scoop bucket add r-bucket https://github.com/cderv/r-bucket.git
+```
+
+Then you can install **one of** the binaries: 
+
+* Recommanded installation of `TinyTeX.zip`
+```powershell
+scoop install TinyTex
+```
+
+* Minimal installation (infra-only) of `TinyTeX-0.zip`
+```powershell
+scoop install TinyTex-min
+```
+
+* Full installation (more packages) of `TinyTeX-1.zip`
+```powershell
+scoop install TinyTex-min
+```
```

Comment:
Let's only provide `TinyTeX-1.zip` for now, and see what users think. If necessary, we could replace it with `TinyTeX.zip` in the future.

Reply:
The doc was incorrect here :) 

`tinytex` scoop package will provide TinyTeX-1.zip